assets/js/timetree.js

```diff
@@ -216,17 +216,75 @@ function TreeGraph({ nodes, links, centuries }) {
   // let width = 1550;
   // let height = 1080;
 
-  let width = 1200; // TODO: can we resize to 800 width for mobile? (tablet portrait and mobile only)
+  // create a media query element to check for mobile / desktop
+  // using breakpoint-s (tablet in portrait orientation)
+  const mobileQuery = window.matchMedia("(max-width: 768px)");
+
+  function isMobile() {
+    return mobileQuery.matches; // true if our query matches
+  }
+
+  function svgWidth() {
+    return isMobile() ? 800 : 1200;
```

Comment:
Are these params somewhere?

Reply:
No, I had it hard-coded in a variable and we wanted to set a different width for mobile than desktop. I could move them into a settings variable similar to what I did with the force strengths; would that be better for documentation / readability?